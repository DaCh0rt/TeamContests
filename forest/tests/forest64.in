833 570
-4095 7046
783 -9249
-8382 4952
355 3270
1891 7679
-1293 -1791
2082 -6716
-3297 6524
-6480 -2605
7919 -5529
2151 -8955
-114 -8928
-1364 3789
-1802 2600
-9607 1985
-1841 1047
-477 -4108
-5016 5001
9098 -343
224 1999
-9301 -4566
-148 -8709
-8355 -7987
8678 -9570
-513 1629
-4748 5500
5489 1633
-7787 -7861
7036 -2250
9193 6523
-3424 4691
-3568 -1292
-6128 1078
-1103 -6228
4170 -5948
4158 48
-9624 78
7871 -3984
-1260 -288
4791 -5419
1732 -7693
6288 5738
9211 769
-6406 -3123
113 -4477
2873 2591
6064 2233
-3362 6171
9443 8873
-4726 -7227
-6943 7831
5302 -4998
5298 -6307
-9814 -7608
794 -7293
-2257 3640
1667 6187
-7038 -1307
3472 1487
-6389 6945
-264 5761
5012 7251
8758 -8060
6023 -6228
-2083 -8578
-7953 3104
6889 9049
-3112 6555
4455 -1260
9276 -9287
9276 -7229
-3529 -1867
885 43
9011 9979
-3389 -9493
9548 -3150
1881 -8350
-7982 -2959
9427 5148
-5792 5326
-9112 7049
-4578 8370
-237 9184
-5423 -7692
-6697 4579
6196 -4385
9521 9997
544 6433
6147 5094
7303 95
5160 157
1749 -3389
6388 -8250
-4051 -6055
6851 -99
3719 4552
4463 -2470
-8862 5497
-3734 -753
-9284 5817
9325 -242
7600 -9501
-2010 -5705
-2805 6666
4586 6301
100 -4706
8822 -7586
-3883 4408
9208 3912
4551 -8146
-2074 -3768
-1215 7617
-7690 -4582
9844 5607
400 -7760
2527 -4742
6989 1234
-9570 -3070
-2889 9654
9317 1083
-6406 4820
-3752 -5021
8134 -888
1727 569
-2605 7766
1382 -2638
380 -8712
1879 9768
-4243 8153
3416 -358
2390 -6680
-5940 193
8926 6618
5609 -5200
-2100 1516
8283 420
2434 5408
1253 -9139
-3324 -9451
6931 5781
-2688 2517
6572 -6362
-893 2633
-603 -3264
-5557 2329
9113 -1564
8463 -5369
-5873 -654
4744 6287
-3183 6002
5260 741
4511 4494
9625 -8501
6437 -7601
-1522 1343
-8023 -7792
9909 9974
-8244 6998
1135 4956
9575 4558
1631 3670
5607 -7904
-3685 -8124
682 -4372
-4108 8439
5617 -250
-2480 2662
-4753 -385
6999 -7288
7061 -8210
-6090 2095
2266 -3233
170 -6691
9458 9721
4874 3532
4102 5353
7837 -3402
7136 650
-441 6608
-5243 -1103
5423 -1562
1714 -2023
-4453 -4893
1218 7370
9588 2937
675 9593
8563 9339
2398 8293
-4262 -4459
4550 -4463
-8738 5769
-3742 576
4399 2972
-3266 3570
3149 6231
6202 6187
-4190 -5893
4055 -8003
5624 -8001
1908 9594
-751 -8927
-81 4734
8399 -24
-7970 7709
5244 5270
9196 -8443
5654 3196
-1106 -8911
-4612 2952
452 -5966
8150 4945
-7579 9777
7190 -1641
-7471 2903
-1055 -1132
-6071 -5950
3129 -7768
3913 -5053
6152 -952
-4589 -9361
977 413
-6091 -6851
-5087 -1095
4714 -2495
-852 8212
-8540 -3321
-8903 499
-5725 4705
3405 9881
6292 7578
2686 1496
7379 1715
-1632 5832
-8530 8490
5011 9298
-1922 -2486
2921 1674
-3097 -12
-3900 -5494
8219 -9503
-171 1198
-3646 -7866
5947 -8876
2293 -1316
8264 -8061
3948 7393
4246 -5152
-2953 -3347
5454 8364
6358 -9966
2869 5804
4617 -7995
8345 131
-493 977
-419 6726
-2312 -1134
7524 6773
742 1455
4985 -6018
-135 -620
6711 5025
1511 -3424
9130 2362
9365 3504
-5373 -5079
-9745 4763
168 5115
-5900 8538
3433 2526
971 -3627
9694 2039
-9609 2946
-2279 -526
125 462
-7998 -3335
-2065 -2703
-904 6284
-9308 -6365
-1872 -3150
-3105 7481
-7773 9985
3614 -7218
8285 -6894
-1480 1017
8799 5212
-4390 3684
-1791 -7559
-9399 9379
-5329 -8019
8473 1568
-7324 -8785
-9548 -6447
-8581 -9546
5789 -7276
1970 -9936
-4352 -1434
4290 -9823
-8127 -1467
6031 -9985
-6016 1100
9265 -9593
1924 6757
8949 -5655
6610 6552
-6733 1696
4925 7405
2858 -8449
2910 1614
6438 -4537
3315 -3389
8901 8234
268 1846
4840 -3976
-7698 9590
-8183 1314
8237 -4982
-123 -8282
-1750 720
-8288 3115
-7311 -2714
-7104 -7869
5011 -3206
6733 8569
-5844 -8381
-43 -3041
8379 7868
-3636 5693
-8726 1388
-9724 -8464
-5429 -4568
4621 3295
9903 -744
9517 -2995
3989 -3041
-5841 4995
1960 -9628
-1800 -6193
-9860 -2809
-9358 3136
-7345 6183
9897 -723
-6649 -3761
-56 6650
-4724 941
-6505 -4589
9204 8095
-2032 6418
-8439 9568
-1501 4359
-1685 -2822
5833 6282
2370 9356
-4586 -1966
3110 -6022
5882 -9078
-4088 575
-7075 -5907
-6858 -3343
9051 -253
8676 -4962
9095 -7456
-8358 9467
-5528 3857
-8409 -8308
7038 -5579
-1741 6082
794 -2423
9169 -2018
-6197 -7350
2581 5435
-1082 1519
-3314 -397
-6402 798
9800 3979
5067 4222
1256 -6490
5486 -487
4511 8038
2408 -8116
9283 3157
2859 -3816
8059 8217
-1560 6096
310 6154
-6292 -5852
-1293 -678
-2565 -15
-3735 -461
-673 -2625
113 590
-5964 2599
-3893 1516
5527 5593
1045 -2590
2300 -5111
9480 2501
5844 -8147
4466 -1628
-3909 -2130
-7446 -615
-3644 675
-1176 5435
-4594 1565
5164 -8918
-6977 -6289
4844 -6379
-9942 9320
2026 -772
-2950 -5994
-2354 -5977
-4245 -5190
-6754 4323
-4025 8450
-7323 -6697
8017 5856
7901 -2028
9589 9989
-6557 -8566
-1355 3927
-6530 9437
-8872 -9558
-8268 -828
8238 5858
-9488 -8201
-8835 3531
8358 -9048
-2388 3251
8440 -540
-1353 -3547
-5970 -7599
7609 -7636
-7329 3410
-301 -6414
392 -9293
110 -8537
6234 -2138
-7864 -1417
-5480 1727
-2911 2640
6875 7399
5041 6193
-4683 5065
-6358 -1364
-8181 3245
208 -636
1670 -1808
4824 -5116
4313 595
6885 -7488
8250 -6334
-8931 -5928
5478 -5298
-5205 773
-9401 128
-6730 -6072
-9842 3785
-4703 2123
5846 -8374
-4274 -8070
8737 8239
4819 1335
1949 -1509
-6278 4362
-6169 -8397
-7063 -8889
-8590 1995
6213 687
9771 -7243
8125 838
9800 -5003
-3870 5565
8639 303
5751 -1036
-2257 4671
5325 8358
-8702 -4897
-7755 6597
3243 -4982
2188 941
-151 -8834
2114 -417
9482 5426
1727 -6389
-9044 -4990
6431 5855
-1841 -4207
7084 -4890
6794 -6804
9900 9637
7738 7104
-5504 3414
9042 -6779
-512 -3305
-355 9374
4678 6905
5587 4460
8806 8158
7935 -5141
4708 -1064
-4784 6341
1271 2685
-565 -1272
9130 -8336
6230 -9813
4454 9755
8655 8048
8157 6079
-3163 7655
-5996 -2700
-748 -8599
2737 2439
7926 -2251
-4130 5711
-3319 1664
1123 -4674
-4680 -2226
8067 8812
8498 5727
3085 2528
4979 -9528
-1353 4618
-6290 6428
7806 -5374
-3380 -5389
9133 960
-4032 9821
-1123 -3912
8144 9748
-8344 -1331
8658 -5661
-510 3912
-1018 6069
2072 -9495
4649 430
-2454 6820
-9420 9451
120 7476
7913 -4305
-7242 -354
-9721 3354
1132 -5399
-637 7675
-2114 -422
7741 -2207
-1054 -494
-8818 5226
-2422 7370
7413 -7480
4436 3501
-5407 6822
3035 1172
-1317 775
3986 -2539
1569 -2531
710 8732
1891 2796
-3164 -6669
5038 -7405
-6383 -8420
7966 9858
-2791 2305
-8788 -2589
1801 -6550
7921 -5675
282 -1332
346 -1108
-9109 4649
-9249 -3407
6235 6571
-7625 3546
-7772 -8044
-2237 7734
1957 -3088
6123 -127
-5049 6523
-3224 4857
8047 9870
5545 6443
1888 1688
-5065 6830
5071 4145
4866 -8292
-9993 13
472 -7421
-7003 -2444
-4537 -2426
8783 -4579
-1915 -7399
-362 -7880
5464 -9832
-6873 -5189
-772 9213
-842 -2998
-653 1867
3246 -7762
3197 -502
3155 -8946
-2723 1424
6348 5188
-4884 -184
-3956 -9953
3595 -9546
-5187 -9260
4650 922
9681 3910
-5260 -8086
3000 5303
225 -4711
-3847 -8513
9795 6984
-6370 174
5907 5150
-2855 -1850
9478 8491
-5827 -6132
4690 -1510
-3835 3881
-9220 688
-5886 5917
-3372 -4899
3992 1659
-594 9262
-4873 9712
7441 4202
2758 -5846
-7371 -3542
-7240 4465
5454 -3046
-4485 3383
5069 1276
-6584 -2996
7765 -5195
1451 -5034
-2841 -7886
-4714 -4421
3676 9471
-2299 -1603
-6485 3342
-196 -8752
4263 -96
4488 -7631
9853 -3298
-1438 8901
-6262 3348
-5395 -7643
-8705 -8413
-4288 -9451
-279 -6383
-3765 -3800
-3736 -7853
2231 2400
813 6932
3962 -5200
-7344 9945
-9856 7637
-628 -5876
-4713 -1967
8008 -2302
-7981 -9379
6063 -7797
-8352 -3446
4304 -8197
7149 9549
-233 -9460
-1115 -5004
9987 5250
-5533 -1016
5412 -2571
-2670 9626
-1179 7320
9844 7817
8314 -1498
1633 -122
1716 -7643
-6485 5282
-953 -994
-3517 -7210
6027 -6158
-4754 1880
9654 -2937
-9113 1998
9611 -8018
3585 -301
-8301 -4755
-1659 7577
5732 7093
6609 8905
-3339 -1777
7906 -5657
8379 4881
-2522 -4788
5583 1534
7165 2988
9953 3274
-9859 -7918
1322 5782
-2285 5232
-4559 6466
6862 -9136
4380 -1915
7515 -7469
-2254 5181
-8193 -7684
-8906 4444
1548 4585
-3483 4678
-4352 -8693
3202 435
-7786 6321
-5746 -9697
-6348 4579
2022 -2920
8214 6022
2892 -3699
-9779 2156
-5312 5136
-9939 857
6235 -5588
1505 -8549
-7332 5747
8275 -5349
8587 -8655
-9003 -8469
-7964 2873
9314 -5215
6503 8510
4605 9416
-2957 9235
-6216 1125
2235 9761
9422 1900
-1059 8475
4750 4890
-9043 -2171
-7448 -8668
5832 840
-4953 -2653
-1902 9980
9699 8561
-2446 -2629
4661 60
5177 -5450
-7323 -9546
-6471 1898
172 7973
-8336 4065
6910 -6118
-7885 2254
-4276 5289
-8030 -9664
-3056 2877
479 -9516
14 -6477
-907 217
7933 2223
9481 9840
-824 -3101
6107 850
3506 884
-3857 7012
6462 389
6906 8767
8556 -2751
1249 -6000
-9478 8962
7269 9094
9053 -8767
7642 -3850
-7366 -4220
6548 -8135
-3059 -6400
-207 6011
-5348 -5581
-8755 4322
7926 7894
-9960 4012
2929 -6943
3186 5323
-2244 814
-5013 1103
-1418 3536
-5057 2191
-5647 -5943
-8054 2242
6949 3175
6175 4490
-5364 9125
9625 1201
4818 3960
-2519 605
-1417 8004
3958 -6692
-1267 -7043
-1826 5964
-9058 -1431
5089 2725
9338 5014
1124 -8619
3394 4649
672 6941
-5486 5385
6518 7921
-7318 -9990
5753 -8273
1676 404
-2755 1888
8912 -5013
3948 -4016
-6949 6650
-3954 -513
7383 9957
-936 2491
-8085 -2358
-4119 3116
-667 -3517
918 1008
7560 -9885
8555 -5294
-9961 -8585
-246 2734
-8713 2781
6599 9639
-8174 -9029
-3031 -6732
-1497 8659
715 1597
-8863 1411
-9379 2841
-908 -1989
1437 8864
-9482 -2534
-4341 9303
2695 -6938
