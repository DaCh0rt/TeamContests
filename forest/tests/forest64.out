80758.17 438513776.99
